server/asset/dcr/dcr.go

```diff
@@ -266,6 +267,20 @@ func (dcr *Backend) FundingCoin(coinID []byte, redeemScript []byte) (asset.Fundi
 	return utxo, nil
 }
 
+// ValidateXPub validates the base-58 encoded extended key, and ensures that it
+// is an extended public, not private, key.
+func (dcr *Backend) ValidateXPub(xpub string) error {
+	xp, err := hdkeychain.NewKeyFromString(xpub, chainParams)
```

Comment:
Decide against a driver interface function since the chainParams is set appropriately by `NewBackend>loadConfig`, so unless you've got a backend, you're stuck with the default chainParams.